app/routes/organen/orgaan/mandaten.js

```diff
@@ -8,10 +8,39 @@ export default class OrganenOrgaanMandatenRoute extends Route {
     const { currentBestuursorgaan } = this.modelFor('organen.orgaan');
 
     const mandaten = await currentBestuursorgaan.bevat;
+    const [orderedMandaten, availableBestuursfuncties] = await Promise.all([
+      this.orderMandaten(mandaten),
+      this.computeBestuursfuncties(mandaten),
+    ]);
 
     return {
       mandaten,
+      orderedMandaten,
       currentBestuursorgaan,
+      availableBestuursfuncties,
     };
   }
+
+  async orderMandaten(mandaten) {
+    return mandaten.slice().sort((a, b) => {
+      return a.get('bestuursfunctie.label') > b.get('bestuursfunctie.label')
+        ? 1
+        : -1;
+    });
+  }
+
+  async computeBestuursfuncties(mandaten) {
+    const bestuursFunctiesUsed = await Promise.all(
+      mandaten.map((m) => m.bestuursfunctie)
+    );
+    const allBestuursfuncties = await this.store.query('bestuursfunctie-code', {
+      page: {
+        size: 200,
+      },
+    });
+    const availableBestuursfuncties = allBestuursfuncties.filter(
+      (bf) => !bestuursFunctiesUsed.includes(bf)
```

Comment:
Not sure why this is a requirement

Reply:
so you don't add duplicate codes (e.g. 2 burgemeesters)